index.js

```diff
@@ -3,3 +3,16 @@ const btn = document.querySelector('button');
 btn.onclick = () => {
     alert('This Button Works!')
 }
+
+window.addEventListener("load", (event) => {
+    console.log("page is fully loaded")
+    fetch("http://api.weatherapi.com/v1/current.json?key=f21c22a7558242f0ac7151418240506&q=37.91822293084342, 21.745074031336777&aqi=no").then(resp=>resp.json()).then(data=>{
```

Comment:
## 🚨 **NEVER** commit your api keys 🚨

Don't worry, I let you do this on purpose to learn 😉 This is a very important lesson that every dev learns someday. Here is your chance to learn the easy way, so pay attention.

This url contains your api key `key=XXXXX`, think of an api key much like a password. It gives you complete access to use the api in your name, this means if anyone were to find this key (and it's pretty easy given this is a public github) they can use the api in any way they see fit. In this case since you haven't paid for the api, the worst case is that your account is shutdown. 

However, in a real project, often times an api key can be for a paid api and someone could snag yours and rack up your bill pretty quick, or it could give them access to your accounts or other user accounts depending on the api. 

So what do we do then?

## Contain the risk
First things first, since this key was committed to the repo, it is now considered compromised. Even if you delete the branch it's still considered a risk. So what we do is "revoke" the key. 

Go to the weather api where you got the api key and revoke it (turn off). 

## How to do this properly
Developers solve this problem by using [environment variables](https://en.wikipedia.org/wiki/Environment_variable). This is fairly easy to do when using a framework but since this project is in vanilla javascript and html, I would recommend creating a separate config file where you store the `key` as a variable and then be sure to add the `config.js` to a [`.gitignore`](https://git-scm.com/docs/gitignore) file as to ensure it never gets improperly committed to the repo.